src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -112,12 +115,12 @@ private[bigquery] class DirectBigQueryRelation(
   }
 
   /**
-   * The theoretical number of Partitions of the returned DataFrame.
-   * If the table is small the server will provide fewer readers and there will be fewer
-   * partitions. If all partitions are not read concurrently, some Partitions will be empty.
-   */
+    * The theoretical number of Partitions of the returned DataFrame.
```

Comment:
nit: on formatting. I don't really care about 2 vs 4 spaces, but I try to follow Databricks style convention of javadoc style instead of scaladoc style (it's a check box in intellij scala style).

You can run sbt scalastyle and sbt test:scalastyle to check. I used to have them block the build, but they blocked compiler messages.

Reply:
I reconfigured my IntelliJ to use JavaDoc style and changed continuations to 4.

sbt scalastyle and sbt test:scalastyle don't report any problems. In some places, I also fixed continuations.

Maybe we can figure out a way to share an IntelliJ style config or have a binary style things automatically. Apache Beam uses spotless, which works pretty well.